code/hud/hudtargetbox.cpp

```diff
@@ -519,7 +519,7 @@ void HudGaugeTargetBox::renderTargetSetup(vec3d *camera_eye, matrix *camera_orie
 	// account for gauge RTT with cockpit here --wookieejedi
 	float clip_aspect;
 	if (gr_screen.rendering_to_texture != -1) {
-		clip_aspect = (clip_width / clip_height);
+		clip_aspect = ( i2fl(clip_width) / i2fl(clip_height) );
```

Comment:
FYI, no spaces needed around the expression. Also, the parenthesis are also not really needed.